core/src/main/java/ch/unibas/dmi/dbis/polyphenydb/rex/RexExecutorImpl.java

```diff
@@ -124,9 +128,18 @@ private String compile( RexBuilder rexBuilder, List<RexNode> constExps, RexToLix
                         ImmutableList.of( root0_ ),
                         blockBuilder.toBlock() );
         String code = Expressions.toString( methodDecl );
-        if ( PolyphenyDbPrepareImpl.DEBUG ) {
+        if ( RuntimeConfig.DEBUG.getBoolean() ) {
             Util.debugCode( System.out, code );
         }
+        if ( dataContext != null && dataContext.getTransaction() != null && dataContext.getTransaction().isAnalyze() ) {
+            InformationManager queryAnalyzer = dataContext.getTransaction().getQueryAnalyzer();
+            InformationPage page = new InformationPage( "informationPageGeneratedCode", "Generated Code" );
+            InformationGroup group = new InformationGroup( page, "Generated Code" );
+            queryAnalyzer.addPage( new InformationPage( "informationPageGeneratedCode", "Generated Code" ) );
+            queryAnalyzer.addGroup( group );
+            InformationCode informationCode = new InformationCode( group, code );
+            queryAnalyzer.registerInformation( informationCode );
+        }
```

Comment:
Please explain this change, what it does, in the Pull Request.

Reply:
This is only for debugging purposes, but I will add a comment.